torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -115,7 +115,10 @@ std::pair<IValue, c10::optional<IValue>> getFunctionTuple(
 
   Inline(*graph);
 
+  // TODO: to use it for mobile, you need to enable it here.
+  // torch::jit::Code code(graph, func.name(), true);
```

Comment:
To enable it, we can consider making a subclass of `Code` that's for mobile only.